src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -69,13 +69,21 @@ See the accompanying LICENSE file for applicable license.
     </delete>
   </target>
   
-  <target name="ditaval-merge" unless="skip.ditaval.preprocess"
-    dita:depends="{depend.preprocess.ditaval-merge.pre}"
+  <target name="ditaval-merge.init">
+    <condition property="skip.ditaval.preprocess" value="true">
```

Comment:
That's ... a good point. It's there because we've always had it as part of DITAVAL. Right now, if you try to call the merge step without having an `args.filter` parameter, I think it fails, but that could be fixed by updating the code to do nothing when `args.filter` isn't specified.

The main thing it would activate, apart from filtering, is that it can turn off flagging. But I'm not sure if that makes sense. And I'm not sure it's valid to say "I want to use this for filtering but not flagging", or the reverse.

Possibly - you could want to turn off flagging pre-process for a specific transform type. But I don't know that this should be grouped in with *all* the preprocess.